### DISCUSSION TITLE
Google Sheets integration

### DISCUSSION THREAD

**Discussion by IgorRSU** [YetiForceCompany/YetiForceCRM#15574] [How to]
Hi, is it possible to integrate YF with Google Sheets? For example, I vape custom Module with 5 fields. So when I'm creating record in my module, I want to insert values from my fields to new row in Sheets.
Thank

**Comment by mariuszkrzaczkowski** [comment]
Yes, there is a lot of information about it, see:
https://developers.google.com/sheets/api/quickstart/php
https://www.srijan.net/resources/blog/integrating-google-sheets-with-php-is-this-easy-know-how
https://pocketadmin-tech.translate.goog/en/version-4-of-the-google-sheets-api-in-php/?_x_tr_sl=en&_x_tr_tl=pl&_x_tr_hl=pl&_x_tr_pto=sc
https://github.com/mach3/google-spreadsheet.php

**Comment by apcloic** [comment]
Maybe this can help
#13933
